overlord/devicestate/devicestate.go

```diff
@@ -637,3 +637,48 @@ func Remodeling(st *state.State) bool {
 	}
 	return false
 }
+
+type recoverySystemSetup struct {
+	// Label of the recovery system
```

Comment:
the comment don't make very clear what are inputs and outputs/state here? Below I see only a couple of these being set

Reply:
Added more information in the comments.